tests/integration/components/tag-post-list-test.js

```diff
@@ -0,0 +1,75 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+const POST = { id: 'why-ember', title: 'Why Ember' };
+
+const POSTS = [
+  POST,
+  { id: 'managing-users', title: 'Managing Ghost users' },
+  { id: 'setting-up-theme', title: 'Setting up your own Ghost theme' },
+  { id: 'advanced-markdown', title: 'Advanced Markdown tips' }
+];
+
+const TAG = {
+  id: 'getting-started',
+  name: 'Getting Started',
+  posts: POSTS
+};
+
+module('Integration | Component | tag-post-list', function(hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.owner.setupRouter();
+  });
+
+  test('it renders a tag name', async function(assert) {
+    this.set('tag', TAG);
+
+    await render(hbs`<TagPostList @tag={{tag}} />`);
+
```

Comment:
can you drop a percy snapshot in here? I think it would be useful to be able to see what the component looks like 👍

Reply:
Sure thing, that's a great idea!

Working on this, I've found out one thing that might be important. Interestingly, a Percy snapshot is slightly different from the component rendered in the app.

### App

<img width="275" alt="Screen Shot 2020-05-05 at 11 34 13 pm" src="https://user-images.githubusercontent.com/5205189/81072152-0f384b00-8f29-11ea-89a2-5c4da3c6f489.png">

### Percy Snapshot

<img width="233" alt="Screen Shot 2020-05-05 at 11 34 44 pm" src="https://user-images.githubusercontent.com/5205189/81072155-11020e80-8f29-11ea-8e75-e29f209071c9.png">

This happens because some `ember-styleguides` styles for links are scoped to the `<main>` element:
https://github.com/ember-learn/ember-styleguide/blob/master/addon/styles/global.css#L158

I'll add `text-decoration: none;` to the `TagPostList` styles to make it render correctly regardless of its context.

@mansona , please, let me know if you have any concerns.

Reply:
Fixed.

Reply:
interesting 🤔 but yes, thanks for catching this 🎉